src/view/utils/useExtensionBridge.js

```diff
@@ -12,8 +12,8 @@ governing permissions and limitations under the License.
 import { useRef, useEffect } from "react";
 
 export default currentViewMethods => {
-  const viewMethods = useRef();
-  viewMethods.current = currentViewMethods;
+  const viewMethodsRef = useRef();
```

Comment:
I always forget I'm dealing with a ref, so I'm trying to suffix all ref variables with `Ref`.